libraries/WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -38,6 +38,12 @@ class FunctionRequestHandler : public RequestHandler {
 
         return true;
     }
+    bool canRaw(String requestUri) override {
+        if (!_ufn || _method == HTTP_GET)
+            return false;
```

Comment:
```suggestion
        if (!_ufn || _method == HTTP_GET) {
            return false;
        }
```

Reply:
What's the problem with that request @nathannau ?